bloggies_backend/src/routes/bookmarks.ts

```diff
@@ -1,42 +1,42 @@
 import { NextFunction, Request, Response } from "express";
 import { ensureLoggedIn } from "../middleware/auth";
 import express from "express";
-import Favorite from "../models/favorite";
+import Favorite from "../models/bookmarks";
 
-export const favoritesRouter = express.Router();
+export const bookmarksRouter = express.Router();
 
-/** GET /favorites/:uid - retrieve favorited posts for a user.
+/** GET /bookmarks/:uid - retrieve favorited posts for a user.
  * Returns a list of posts */
-favoritesRouter.get("/:uid", async function (req: Request, res: Response, next: NextFunction) {
+bookmarksRouter.get("/:uid", async function (req: Request, res: Response, next: NextFunction) {
   const userId = parseInt(req.params.uid);
   try {
-    const posts = await Favorite.getAllFavorites(userId);
+    const posts = await Favorite.getAllBookmarks(userId);
```

Comment:
Favorite.getAllBookmarks => Bookmark.getAllBookmarks

(same for more on this page)